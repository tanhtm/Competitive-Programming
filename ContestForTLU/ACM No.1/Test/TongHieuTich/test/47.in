60
699 392
-592 -894
-162 -895
-275 -165
-81 -50
842 762
39 217
-573 -503
-894 196
-442 813
-708 -557
73 -621
22 493
606 -893
110 -249
751 756
304 884
49 -509
417 -372
345 -290
134 935
827 -315
-916 -581
-688 330
566 -847
707 785
-157 161
-802 606
-540 150
-465 -112
44 327
-657 -59
257 160
297 907
419 49
654 -170
-320 -532
995 -565
244 -263
673 546
57 659
917 680
728 109
-279 -766
649 -837
754 303
-917 -153
-698 720
730 -465
-144 264
372 -591
356 -529
717 -387
-689 -32
536 -713
-784 120
-936 150
413 194
-784 608
-488 -737
