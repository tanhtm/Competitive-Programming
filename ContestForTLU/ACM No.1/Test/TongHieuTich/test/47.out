83734
-112193
-193695
12100
1015
32080
-11392
18830
190205
-116401
47753
-95078
-60641
-107553
-12475
-1883
-172260
-64170
8876
8731
-214067
146176
125373
91111
-99263
-29094
-318
68992
67275
50920
-26248
107042
10112
-183610
43290
99704
-45156
167700
-2408
38703
-107758
94622
129525
-127228
-69842
119176
204370
-7799
79168
-12240
-52724
-38276
91080
118424
-55268
150064
213399
33233
61248
-76256
